Fix compilation error that happened with 32 bit arm:
/dom/media/webrtc/libwebrtcglue/WebrtcGmpVideoCodec.cpp:415:10: error: non-constant-expression cannot be narrowed from type 'uint64_t' (aka 'unsigned long long') to 'const unsigned int' in initializer list [-Wc++11-narrowing-const-reference]

--- ./dom/media/webrtc/libwebrtcglue/WebrtcGmpVideoCodec.cpp	2024-04-23 16:37:04.543077350 +0200
+++ ./dom/media/webrtc/libwebrtcglue/WebrtcGmpVideoCodec.cpp	2024-04-23 16:39:02.877063991 +0200
@@ -412,7 +412,7 @@
     auto inputImageMap = aEncoder->mInputImageMap.Lock();
     DebugOnly<bool> inserted = false;
     std::tie(std::ignore, inserted) = inputImageMap->insert(
-        {frame->Timestamp(), {aInputImage.timestamp_us()}});
+        {(uint32_t)frame->Timestamp(), {aInputImage.timestamp_us()}});
     MOZ_ASSERT(inserted, "Duplicate timestamp");
   }
 
